src/health.controller.ts

```diff
@@ -1,9 +0,0 @@
-import { Controller, Get } from '@nestjs/common';
-
-@Controller('health')
-export class HealthController {
```

Comment:
**issue:** please keep the `HealthController`, there is no need to delete it

Reply:
Fixed: Added it back.